### DISCUSSION TITLE
WW 12 - Noam

### DISCUSSION THREAD

**Discussion by NoamRahat** [FPGA-MAFIA/fpga_mafia#578] [Weekly]
# Completed:
- #546
  - I have ensured proper handling of `scanf()` for keyboard input, including parsing strings into integers.
  - Encapsulated program logic into functions for modularity and potential reuse.
  - Developed test logic in `verif/core_rrv/tests/PS2_alive.c` and defined keyboard input sequences in `verif/core_rrv/tests/PS2_alive.vh`.
  - Utilized `send_string()` command in the .vh file for simulating keyboard input.
  - Added new tests to the test sequence file `at verif/core_rrv/tb/hw_seq/core_rrv_hw_seq.vh`.
  - Used the `-top core_rrv_no_ref_tb` option to simplify verification during test execution.

# WIP:
- #553

# Not Started:
- #568 

## _I will be OOO until 10.4 deu to exam period._

**Comment by amichai-bd** [comment]
Thanks for the update!
If you already WIP on the #553 please let me know if you have any questions.

Good luck with the exams!
